libs/template/presets/storefront/experience/pages/home-page.ts

```diff
@@ -1,19 +1,23 @@
-import { StaticComponent, StyleRuleSet } from '@spryker-oryx/experience';
+import { ExperienceComponent, StyleRuleSet } from '@spryker-oryx/experience';
 
 const brand = (name: string, rules?: StyleRuleSet[]) => ({
   type: 'oryx-content-image',
   name,
-  content: { graphic: name, label: name, link: `/search?q=${name}` },
+  content: { data: { graphic: name, label: name, link: `/search?q=${name}` } },
   options: { rules },
 });
 
-export const homePage: StaticComponent = {
+export const homePage: ExperienceComponent = {
+  id: 'home',
   type: 'Page',
   meta: {
     title: 'Home Page',
     route: '/',
     description: 'Home Page Description',
   },
+  ref: {
+    header: 'prepend',
```

Comment:
A ref is always added at a specific location and should nit need a merge strategy